docs/source/configuration/how-to.md

```diff
@@ -46,9 +46,6 @@ You must provide a function as default export in your `src/config.js`:
 export default function applyConfig(config) {
   config.settings = {
     ...config.settings,
-    isMultilingual: true,
```

Comment:
OMG... probably the first config that we had in plone-react/Volto ever... getting rid of it is a thing!